website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -54,3 +67,61 @@ Options:
   -h, --help                           Print help
   -V, --version                        Print version
 ```
+
+### Split DNS
+
+The Linux Client supports Split DNS, which you can enable by setting the
+`FIREZONE_DNS_CONTROL` environment variable to `systemd-resolved` or
+`etc-resolv-conf`. If you don't set this variable, Split DNS will be disabled.
```

Comment:
Yeah let's make systemd-resolved the default since it's safe. Modifying resolv.conf feels risky on non-cattle systems, so maybe that could be opt-in.

https://github.com/firezone/firezone/issues/3812